### PR TITLE
Add nutanix disconnected ci job for 4.12

### PR DESCRIPTION
**Pull request by sgaoshang**

### CODE REVIEW

**Comment by jianlinliu** [comment]
cc @liangxia to review.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -2603,31 +2603,80 @@ tests:
...
-- as: nutanix-ipi-fips-p2-f7
+- as: nutanix-ipi-p2-f28
   cluster: build01
-  cron: 57 16 2,10,17,24 * *
+  cron: 44 21 10 * *
   steps:
     cluster_profile: nutanix-qe
     env:
       E2E_RUN_TAGS: '@amd64 and @nutanix-ipi and @network-ovnkubernetes'
-      FIPS_ENABLED: "true"
```
[the remark this answers is not in the document]
Comment (sgaoshang) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:2613]: fixed, pls review again, thanks.

**Pull request merged**